src/js/components/SharedComponents/SubmissionWarningBanner.jsx

```diff
@@ -51,15 +51,17 @@ export default class SubmissionWarningBanner extends React.Component {
                 unpublished: null,
                 'test-submission': {
                     header: 'This is a test submission',
-                    message: 'Test submissions cannot be published or certified, but they can be used to validate your data.',
+                    message: `Test submissions cannot be published or certified, but they can be used to validate ` +
+                        `your data.`,
                     useMarkdown: true
                 },
                 'test-submission-one': {
                     header: (<p>This is a test submission</p>),
                     message: (
                         <p>
                             A submission has already been published and/or certified for this time period.
-                            To view the published and/or certified submission, <Link to={`/submission/${pubSub}/`}>click here</Link>.
+                            To view the published and/or certified submission,
+                            <Link to={`/submission/${pubSub}/`}>click here</Link>.
```

Comment:
There's supposed to be a space between "certified submission," and "click here" that gets removed with this formatting.